src/app/pages/system/view-enclosure/components/set-enclosure-label-dialog/set-enclosure-label-dialog.component.ts

```diff
@@ -21,7 +21,7 @@ export interface SetEnclosureLabelDialogData {
 })
 export class SetEnclosureLabelDialogComponent implements OnInit {
   form = this.formBuilder.group({
-    label: ['', Validators.required],
+    label: ['', [Validators.required, Validators.pattern('^(?!\\s*$).+')]],
```

Comment:
Let's use `IxValidatorService.withMessage` method to give a readable message for this pattern validator.